### DISCUSSION TITLE
Error while verify Verify domain(s)

### DISCUSSION THREAD

**Discussion by amaalrgb** [reown-com/appkit#3163] [Q&A]
I am trying to verify a domain like `*.stage.company.com`, I've added the Verification code as TXT record on my dns provider as per the doc against `root(@)` and `*.stage`. After that I've tried the txt record is working via mxtoolbox and after I click verify on the page it is giving me error `None of the provided domains passed verification`. When I check the api call response from the network tab I am seeing the error `Bad request - homepage must be a valid URL`. Can anyone help me here

**Comment by amaalrgb** [comment]
This is fixed, they were not supporting wildcard domains but the error was not remotely close to what the actual issue was.

**Comment by pad01g** [comment]
@amaalrgb 
Hi, did you successfully verify your subdomain?
In my case, I could only verify `company.com` even when I wanted to register `stage.company.com` domain.

Also, it's strange that I have to verify domain for my web wallet project.
I can understand that dapps projects need to verify domain (to prevent scams), but I'm not sure why I need to verify domain for wallet. Our wallet product is expected to be used in many websites context (like web3auth), and we don't expect every website owners register their domains in walletconnect website.

**Reply by pad01g** [reply]
Never mind, I created a separate question: https://github.com/orgs/reown-com/discussions/3459
